qa/L0_batcher/test.sh

```diff
@@ -39,64 +39,75 @@ CLIENT_LOG="./client.log"
 BATCHER_TEST=batcher_test.py
 VERIFY_TIMESTAMPS=verify_timestamps.py
 
-DATADIR=/data/inferenceserver/${REPO_VERSION}
+DATADIR=${DATADIR:="/data/inferenceserver/${REPO_VERSION}"}
+OPTDIR=${OPTDIR:="/opt"}
+SERVER=${OPTDIR}/tensorrtserver/bin/trtserver
 
-SERVER=/opt/tensorrtserver/bin/trtserver
 source ../common/util.sh
 
 RET=0
 
+# If BACKENDS not specified, set to all
+BACKENDS=${BACKENDS:="graphdef savedmodel netdef onnx libtorch plan custom"}
+
 # Must run on a single device or else the TRTSERVER_DELAY_SCHEDULER
 # can fail when the requests are distributed to multiple devices.
 export CUDA_VISIBLE_DEVICES=0
 
 # Setup non-variable-size model repository
 rm -fr *.log *.serverlog models && mkdir models
-for m in \
-        $DATADIR/qa_model_repository/savedmodel_float32_float32_float32 \
-        $DATADIR/qa_model_repository/graphdef_float32_float32_float32 \
-        $DATADIR/qa_model_repository/netdef_float32_float32_float32 \
-        $DATADIR/qa_model_repository/plan_float32_float32_float32 \
-        $DATADIR/qa_model_repository/libtorch_float32_float32_float32 \
-        $DATADIR/qa_model_repository/onnx_float32_float32_float32 \
-        ../custom_models/custom_float32_float32_float32 ; do
-    cp -r $m models/. &&
-        (cd models/$(basename $m) && \
-                sed -i "s/^max_batch_size:.*/max_batch_size: 8/" config.pbtxt && \
-                sed -i "s/^version_policy:.*/version_policy: { specific { versions: [1] }}/" config.pbtxt && \
-                echo "dynamic_batching { preferred_batch_size: [ 2, 6 ], max_queue_delay_microseconds: 10000000 }" >> config.pbtxt)
+for BACKEND in $BACKENDS; do
+    if [ "$BACKEND" != "custom" ]; then
+      cp -r $DATADIR/qa_model_repository/${BACKEND}_float32_float32_float32 models/. &&
+        (cd models/${BACKEND}_float32_float32_float32 && \
+            sed -i "s/^max_batch_size:.*/max_batch_size: 8/" config.pbtxt && \
+            sed -i "s/^version_policy:.*/version_policy: { specific { versions: [1] }}/" config.pbtxt && \
+            echo "dynamic_batching { preferred_batch_size: [ 2, 6 ], max_queue_delay_microseconds: 10000000 }" >> config.pbtxt)
+    else
+      cp -r ../custom_models/custom_float32_float32_float32 models/. &&
```

Comment:
The only different is the copy path, you can just condition on the `cp` command, the rest is the same.